release/models/aft/openconfig-aft-common.yang

```diff
@@ -244,11 +258,134 @@ submodule openconfig-aft-common {
           }
         }
 
+        container encap-headers {
```

Comment:
How does decapsulation work here?

For example, we have `pop-top-label` which is a decapsulation action. We are leaving `decapsulate-header` in place un-changed. So does this mean that we should leave anything that is doing decapsulation in the same place and not change where it lives (this means keeping `pop-top-label` where it is today within `next-hop/state`).

Reply:
I would like to leave the changes to decapsulation  in a follow up PR.  At the moment, the existing OC models meet our operational use case for decapsulation without changes.

`pop-top-label` is still in the same place in the tree.

Reply:
Disagree re: `pop-top-label` is in the same place in the tree. This change makes `pop-top-label` be available multiple times in the list. If it's solely in the same place in the tree then it would just be at the state location before. 

So, either we clarify how this works here, or we leave `pop-top-label` at exactly the same place is and was before, and *not* here.

Reply:
Ah, you're right, it was added to the encap-header/mpls container, where it does not belong.  I have removed it.